melodies_monet/driver.py

```diff
@@ -394,6 +418,13 @@ def open_model_files(self, time_interval=None):
                 self.obj = mio.raqms.open_mfdataset(self.files,**self.mod_kwargs)
             else:
                 self.obj = mio.raqms.open_dataset(self.files,**self.mod_kwargs)
+        elif time_chunking_with_gridded_data:
+            date_str = time_interval[0].strftime('%Y-%m-%b-%d-%j')
+            print('model time chunk %s' % date_str)
+            model_datasets, filenames = read_grid_util.read_grid_models(
+                control_dict, date_str, model=self.label)
+            print(filenames)
+            self.obj = model_datasets[self.label]
```

Comment:
Wouldn't you want this if statement to occur prior to all of the other model options? I'm imagining if I were to try to use `time_chunking_with_gridded_data ` but `mod_type=='wrfchem'` or something else, the code would never make it to this `elif` statement.

If I'm understanding the purpose correctly, you would want to have this set up like follows (similar to in the `driver.observation.open_obs()` section):

````
if time_chunking_with_gridded_data:
    run some code...
else:
    if 'cmaq' in self.model.lower():
        run some code
    elif 'wrfchem' in self.model.lower():
        run some code
    ........

````